core/ecschema-metadata/test/UnitConversion/cross-schema-test-data.json

```diff
@@ -0,0 +1,322 @@
+[
```

Comment:
these json files should also go into the assets directory, same as the test schemas